### ISSUE TITLE
Update the theme styles of the website

### ISSUE THREAD

**Issue by frank-zsy** [X-lab2017/open-digger-website#42]
### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Use case description

I would like to update the theme color of the website as well as the default styles of charts and table components.

**Comment by Neilblaze** [comment]
@frank-zsy I am trying to understand the issue, since we already have a toggle-btn between **light/dark** mode, I'm curious to know what is meant by *update of the theme color* of the website 🤔 

Is it something connected to [this](https://docusaurus.io/docs/api/themes/configuration)?

**Comment by frank-zsy** [comment]
Yes, actually I found a UI designer to give us a new design of our color theme and especially the chart styles. I will do this and the preview looks like:

<img width="707" alt="image" src="https://github.com/user-attachments/assets/125d35b2-bc12-41a0-a739-ad3311bc43ec">

<img width="707" alt="image" src="https://github.com/user-attachments/assets/e061ccb9-1761-4022-902d-19ee407885e8">

<img width="704" alt="image" src="https://github.com/user-attachments/assets/ad6dc1f3-afb5-4ef5-abc6-0472f657fa8f">

The charts background color will stay identical with the website background color and I will optimize the options for charts and inject default styles for line, bar charts, WDYT?

**Comment by Neilblaze** [comment]
@frank-zsy I loved it! 😃 

Just curious, is the color theme inspired by both [OpenAtom](https://openatom-dashboard.x-lab.info/) & [PolarDB](https://dataease.x-lab.info/link/RyaY6hry) dashboard (themes)? 🤔  Nevertheless, the UI components look much more consistent across the pages. For the charts, I'd recommend aiming for high-contrast colors (as shown in the preview) to maximize the page's aesthetics. 

And yes, let's go for this! 🚀

**Comment by frank-zsy** [comment]
@Neilblaze Not quite inspired by OpenAtom & PolarDB dashboard. Actually I am personally prefer deep blue color theme which has a stronger sense of technology.😄
